src/utils/discordUtil.ts

```diff
@@ -54,9 +55,9 @@ export const addRoleToUser = async (userId: string, guild: Guild | null, roleNam
  * @returns {Promise<any>}
  */
 export const fetchPrefix = async (message: Message) => {
-  const {guildId} = message;
+  if (!message.guild) return DEFAULT_PREFIX;
 
-  const guildPrefix = (await getSettings(guildId as string)).prefix;
+  const {guildId} = message;
 
-  return guildPrefix ?? '$';
+  return (await getSettings(guildId as string)).prefix;
 };
```

Comment:
Please test this function

Reply:
I thought we agreed not to test stuff that's through discord cause we'd have to mock messages and stuff?

Reply:
The only part of this function that is not really discord related is getSettings, which has its own tests